packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -88,6 +94,24 @@ export class ForceFunctionCreateExecutor extends BaseTemplateCommand {
       });
     }
   }
+
+  public logMetric(
+    logName: string | undefined,
+    hrstart: [number, number],
+    properties?: Properties,
+    measurements?: Measurements
+  ) {
+    if (properties) {
+      const fileExtension = this.getFileExtension().replace('.', '');
+      if (fileExtension === 'js') {
+        properties.language = 'node';
+      } else if (fileExtension === 'java') {
+        properties.language = 'java';
+      }
```

Comment:
Not sure if we should limit only reporting the language if it's one of these two things, maybe there should be a fallback

Reply:
Can't think of a scenario where a fallback would be required. The metaDictionary definitions does not have a default MetaDataInfo, so good to follow the same pattern.

Reply:
@randi274 - While looking into this, I see in metadataDictionary.ts the suffix is set to blank for functionjs and functionjava. I think we need to take care of this. Have we tested this end to end to see if it works? Testing telemetry in local is a bit of a challenge (you need to use the vsix from circleci). Let me know if you need help with that.

Reply:
@jag-sfdc some help would be great, thank you! I was able to get the vsix files installed, but I'm not quite sure how to run the telemetry locally, and could use some help triggering the functions as well since I had some difficulty getting that set up. I'll set up a meeting for tomorrow 👍